src/Sulu/Bundle/WebsiteBundle/EventListener/AppendAnalyticsListener.php

```diff
@@ -82,16 +102,75 @@ public function onResponse(FilterResponseEvent $event)
         }
 
         $portalUrl = $this->requestAnalyzer->getAttribute('urlExpression');
-        $analytics = $this->analyticsRepository->findByUrl(
+
+        /** @var Analytics[] $analyticsArray */
+        $analyticsArray = $this->analyticsRepository->findByUrl(
             $portalUrl,
             $this->requestAnalyzer->getPortalInformation()->getWebspaceKey(),
             $this->environment
         );
 
-        $content = $this->engine->render('SuluWebsiteBundle:Analytics:website.html.twig', ['analytics' => $analytics]);
+        $analyticsContent = [];
+        foreach ($analyticsArray as $analytics) {
+            $this->generateAnalyticsContent($analyticsContent, $analytics);
+        }
 
         $response = $event->getResponse();
-        $responseContent = $response->getContent();
-        $response->setContent(str_replace('</body>', $content . '</body>', $responseContent));
+        $content = $response->getContent();
+
+        $this->setAnalyticsContent($content, $analyticsContent);
```

Comment:
write: `$response->setContent($this->setAnalyticsContent($content, $analyticsContent));`